databases/week2/part2.sql

```diff
@@ -0,0 +1,25 @@
+USE hyf_class2023;
+
+-- create table Class with the columns: id, name, begins (date), ends (date)
+CREATE TABLE IF NOT EXISTS Class (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(255) NOT NULL,
+  `begins_date` DATETIME NOT NULL,
+  `ends_date` DATETIME NULL DEFAULT NULL,
```

Comment:
columns` begins` &` ends` were supposed to have date not datetime as a datatype.

Reply:
ok.